boudicca.events/enricher/src/main/kotlin/events/boudicca/enricher/service/osm/OsmEnricher.kt

```diff
@@ -17,6 +16,7 @@ import base.boudicca.model.structured.VariantConstants.SOURCE_VARIANT_NAME
 import com.fasterxml.jackson.databind.JsonNode
 import com.fasterxml.jackson.databind.node.ArrayNode
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import events.boudicca.enricher.service.EnricherOrderConstants
 import io.github.oshai.kotlinlogging.KotlinLogging
 import org.springframework.core.annotation.Order
 import org.springframework.stereotype.Service
```

Comment:
why is the OSMEnricher not part of the lib anymore, but now boudicca.events specific? IMO it could be reused by other boudicca applications, not?

Reply:
because i don't think it makes sense to implement enrichers more then once, especially if a lot of infrastructure is needed for that. for example for the OSM Enricher you need your own Nominatim instance, which is something i setup for us but others probably won't like to do. they should just use the osm enricher we are using